bin/setenv-docker-customize.sh

```diff
@@ -175,6 +175,14 @@ EXO_ES_URL="${EXO_ES_SCHEME}://${EXO_ES_HOST}:${EXO_ES_PORT}"
 
 [ -z "${EXO_ADDONS_CONFLICT_MODE}" ] && EXO_ADDONS_CONFLICT_MODE=""
 
+[ -z "${EXO_JCR_DB_STORAGE_ENABLED}" ] && EXO_JCR_DB_STORAGE_ENABLED=""
```

Comment:
```suggestion
[ -z "${EXO_JCR_DB_STORAGE_ENABLED}" ] && EXO_JCR_DB_STORAGE_ENABLED="true"
```
WDYT @boubaker please?